core/src/main/java/nl/nn/adapterframework/extensions/javascript/J2V8.java

```diff
@@ -17,6 +17,8 @@
 
 import java.io.File;
```

Comment:
When you work on a class you have to update the copyright notice. And since we change name `Integration Partners` can be changed with `WeAreFrank!`